### PR TITLE
Add missing mrkd dependency in Ubuntu install docs

### PR DESCRIPTION
**Pull request by sethforprivacy**

### CODE REVIEW

File: doc/INSTALL.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Get dependencies:
       autoconf automake build-essential git libtool libgmp-dev libsqlite3-dev \
       python3 python3-pip net-tools zlib1g-dev libsodium-dev gettext
     pip3 install --upgrade pip
-    pip3 install --user poetry
+    pip3 install --user poetry mrkd
```
Comment (vincenzopalazzo) [inline doc/INSTALL.md:44]: nit: this is not missing, is removed!

What is the reason to add this dependencies this specific case? Do you get some error?

Comment (sethforprivacy) [reply doc/INSTALL.md:44]: I am unable to build from source without it install when checking out v0.12.0 tag:

https://lightning.readthedocs.io/INSTALL.html#to-build-on-ubuntu

Will remove mrkd and re-run to get the exact errors but its still a dependency as of v0.12.0.

Comment (sethforprivacy) [reply doc/INSTALL.md:44]: Logs of the failure can be seen here:

https://paste.sethforprivacy.com/?f02a8a6888174f91#9v34xrZwiwL2Zwa5U9WJu7vcE2EyZZ5Cb5XvGHGsyJTm

The most useful snippet is at the end:

```
mrkd doc/lightning-cli.1.md
/bin/sh: 1: mrkd: not found
make: *** [doc/Makefile:131: doc/lightning-cli.1] Error 127
mkdir -p /usr/local/bin
mkdir -p /usr/local/libexec/c-lightning
mkdir -p /usr/local/libexec/c-lightning/plugins
mkdir -p /usr/local/share/man/man1
mkdir -p /usr/local/share/man/man5
mkdir -p /usr/local/share/man/man7
mkdir -p /usr/local/share/man/man8
mkdir -p /usr/local/share/doc/c-lightning
cp tools/hsmtool tools/lightning-hsmtool
install cli/lightning-cli lightningd/lightningd tools/lightning-hsmtool /usr/local/bin
install lightningd/lightning_channeld lightningd/lightning_closingd lightningd/lightning_connectd lightningd/lightning_dualopend lightningd/lightning_gossipd lightningd/lightning_hsmd lightningd/lightning_onchaind lightningd/lightning_openingd lightningd/lightning_websocketd /usr/local/libexec/c-lightning
[ -z "plugins/autoclean plugins/chanbackup plugins/bcli plugins/commando plugins/fetchinvoice plugins/funder plugins/topology plugins/keysend plugins/offers plugins/pay plugins/txprepare plugins/spenderp plugins/bookkeeper" ] || install plugins/autoclean plugins/chanbackup plugins/bcli plugins/commando plugins/fetchinvoice plugins/funder plugins/topology plugins/keysend plugins/offers plugins/pay plugins/txprepare plugins/spenderp plugins/bookkeeper /usr/local/libexec/c-lightning/plugins
mrkd doc/lightning-cli.1.md
/bin/sh: 1: mrkd: not found
make: *** [doc/Makefile:131: doc/lightning-cli.1] Error 127
```

Comment (sethforprivacy) [reply doc/INSTALL.md:44]: I just pulled source today, checked out v0.12.0, here is commit hash:

```
sethforprivacy in 🌐 apps in lightning (3ce2673 🏷  v0.12.0) is 📦 v0.1.0 via C v11.2.0-gcc via 🦀 took 4s ❯ git rev-parse --verify HEAD
3ce26735e5143fa9532247156f8e97291cb2a225
```

**Comment by sethforprivacy** [comment]
Added an installation of it as sudo, as if you only install as user the `sudo make install` step will fail as it also relies on `mrkd`.

**Comment by cdecker** [comment]
We've removed the dependency on `mrkd` in the repo, since we now commit all the generated files and users should not need to regenerate them anymore. If it is still required we failed in checking these files in, so we'd better amend that commit instead (notice that release tags are always a bit tricky since we can't change them after the fact, so we may have to live with it in this state).

**Comment by sethforprivacy** [comment]
Will close this in that case, thanks for the clarification!

**Pull request closed without merging**